src/modules/Dashboard/components/ActivityAssignDrawer/ActivityCheckbox/ActivityCheckbox.tsx

```diff
@@ -5,7 +5,14 @@ import { variables } from 'shared/styles';
 
 export const ActivityCheckbox = (props: CheckboxProps) => (
   <Checkbox
-    icon={<Svg id="add-circle" fill={variables.palette.on_surface_variant} />}
+    icon={
+      <Svg
+        id="add-circle"
+        fill={
+          props.disabled ? variables.palette.outline_variant : variables.palette.on_surface_variant
+        }
```

Comment:
Left a figma [comment](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x?node-id=10933-129273#957753165) for Matt to confirm this colour for disabled.